executor/nsinit/nsinit.go

```diff
@@ -131,7 +131,12 @@ func (e *Executor) Execute(f def.Formula, j def.Job, d string, result *def.JobRe
 	cmd.Stderr = errS
 
 	// Prepare filesystem
-	util.ProvisionInputs(f.Inputs, rootfs, journal)
+	transmat := util.DefaultTransmat()
+	util.ProvisionInputs(
+		transmat,
+		util.BestAssembler(),
+		f.Inputs, rootfs, journal,
+	)
```

Comment:
You forgot this line:

`defer assembly.Teardown() // What ever happens: Disassemble filesystem`

Exists in chroot but not nsinit.